applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -301,3 +301,94 @@ Example
 
    AT#XSLMUART=?
    #XSLMUART: (1200, 2400, 4800, 9600, 14400, 19200, 38400, 57600, 115200, 230400, 460800, 921600, 1000000)
+
+SLM DATA MODE CONTROL #XDATACTL
+===============================
+
+The ``#XDATACTRL`` command configure size or time limit for data mode.
+
+Set command
+-----------
+
+The set command configures size or time limit for data mode.
```

Comment:
```suggestion
The set command configures a size or time limit for data mode.
```

Reply:
> Does any of the example output in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/doc/slm_description.html#testing need updates to show the pattern string?

This PR does not update previous AT command interface so no change is needed